proto/FeatureCollection/FeatureCollection.proto

```diff
@@ -1,210 +1,256 @@
-﻿/*
-  Copyright 2021 Esri
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-    http://www.apache.org/licenses/LICENSE-2.0
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
-*/
-
-syntax = "proto3";
+﻿syntax = "proto3";

```

Comment:
Do we need the copyright here? We already list in the license file. Keeping this file 1-to-1 with the internal repo should make things easier to keep in sync.